Use after move when window feature is enabled

**Description**
Rust compiler errors due to use after move when trying to compile `game-loop` with `window` feature enabled.

**Reproduction Steps**
```toml
[dependencies]
game-loop = { version = "0.9.0", features = ["window"] }
```

```sh
$ cargo run
```

**Error**
```
   Compiling game-loop v0.9.0
error[E0382]: use of moved value: `event`
  --> /home/ben/.cargo/registry/src/github.com-1ecc6299db9ec823/game-loop-0.9.0/src/helper.rs:77:19
   |
71 |         event_loop.run(move |event, _, control_flow| {
   |                              ----- move occurs because `event` has type `Event<'_, T>`, which does not implement the `Copy` trait
...
75 |             handler(&mut game_loop, event);
   |                                     ----- value moved here
76 | 
77 |             match event {
   |                   ^^^^^ value used here after move
```

--- a/src/helper.rs
+++ b/src/helper.rs
@@ -63,7 +63,7 @@ mod helper {
         where G: 'static,
               U: FnMut(&mut GameLoop<G, Time, Window>) + 'static,
               R: FnMut(&mut GameLoop<G, Time, Window>) + 'static,
-              H: FnMut(&mut GameLoop<G, Time, Window>, Event<'_, T>) + 'static,
+              H: FnMut(&mut GameLoop<G, Time, Window>, &Event<'_, T>) + 'static,
               T: 'static,
     {
         let mut game_loop = GameLoop::new(game, updates_per_second, max_frame_time, window);
@@ -72,7 +72,7 @@ mod helper {
             *control_flow = ControlFlow::Poll;
 
             // Forward events to existing handlers.
-            handler(&mut game_loop, event);
+            handler(&mut game_loop, &event);
 
             match event {
                 Event::RedrawRequested(_) => {

